quilt/data.py

```diff
@@ -20,86 +20,94 @@
 
 from six import iteritems
 
-from .tools.core import GroupNode as CoreGroupNode
+from .tools import core
 from .tools.store import PackageStore
 
 __path__ = []  # Required for submodules to work
 
-class PackageNode(object):
+class Node(object):
     """
     Abstract class that represents a group or a leaf node in a package.
     """
-    def __init__(self, package, prefix, node):
+    def __init__(self):
         # Can't instantiate it directly
-        assert self.__class__ != PackageNode.__class__
+        assert self.__class__ != Node.__class__
 
-        self._package = package
-        self._prefix = prefix
-        self._node = node
-
-    def __eq__(self, other):
-        if isinstance(other, self.__class__):
-            return self._package == other._package and self._prefix == other._prefix
-        return NotImplemented
-
-    def __ne__(self, other):
-        return not self == other
-
-    def __hash__(self):
-        return hash((self._package, self._prefix))
+    def _class_repr(self):
```

Comment:
This is now a totally private method - I wonder if I should use `__` or something to distinguish it from "public" methods that start with `_`?